Framework/script/RepoCleaner/qcrepocleaner/o2-qc-repo-delete-versions-not-in-periods

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python3
+from collections import defaultdict
+import logging
+import argparse
+
+from qcrepocleaner import binUtils
+from qcrepocleaner.Ccdb import Ccdb
+import dryable
+
+def parse_args():
+    """Parse the arguments passed to the script."""
+    logging.info("Parsing arguments")
+    parser = argparse.ArgumentParser(description='Remove all the versions in the given path that don\'t match the given list of periodNames ')
```

Comment:
perhaps you could specify explicitly what will happen to versions without `periodName`.

Reply:
yes indeed.